tests/go-validation/v1apitests/provisioning_test.go

```diff
@@ -0,0 +1,92 @@
+package v1_api_tests
+
+import (
+	"fmt"
+	"testing"
+
+	apisV1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	v1 "github.com/rancher/rancher/pkg/generated/norman/core/v1"
+	"github.com/rancher/rancher/tests/go-validation/rancherrest"
+	"github.com/rancher/rancher/tests/go-validation/utils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	digitalOceanCloudCredentialName = "docloudcredential"
+	namespace                       = "fleet-default"
```

Comment:
do we need to specify the `fleet-default` namespace here? Just curious I know that the new API is using fleet for provisioning but im not familiar how is that working internally.

Reply:
Yes we do, we have specify "fleet-default"